tests/test_c_api/test1.c

```diff
@@ -1945,7 +1954,9 @@ void Tests_B_MODEL()
 
     // Check some results
     S4ASSERT(KV_get_at_aper("UY", "2000Y1") == 650.0, "Exchange UY-XNATY: UY[2000Y1] == 650.0 unchanged");
-    S4ASSERT(U_test_eq(KV_get_at_aper("XNATY", "2000Y1"), 0.8006766), "Exchange UY-XNATY: XNATY[2000Y1] == 0.80069");
+    XNATY_2000Y1 = KV_get_at_aper("XNATY", "2000Y1"); 
+    //printf("XNATY_2000Y1 = %lg\n", XNATY_2000Y1);
+    S4ASSERT(U_test_eq(KV_get_at_aper("XNATY", "2000Y1"), 0.800673), "Exchange UY-XNATY: XNATY[2000Y1] == 0.800673");
```

Comment:
The problem is in `tests/test_cpp_api/test_simulation.cpp`, not `tests/test_c_api`.